photon-api/src/main/scala/com/linkedin/photon/ml/data/RandomEffectDataset.scala

```diff
@@ -392,9 +399,16 @@ object RandomEffectDataset {
       .features
       .length
 
+    val priorModels = priorRandomEffectModelOpt.map(_.modelsRDD).getOrElse(sc.emptyRDD[(REId, GeneralizedLinearModel)])
```

Comment:
I think these changes have been merged by PR-462. Can you merge the conflict and update the PR for an updated diff?

Reply:
updated